src/config.cpp

```diff
@@ -0,0 +1,64 @@
+#include "config.hpp"
+#include "sdcard.hpp"
+
+#include <string_view>
+#include <ext/stdio_filebuf.h> 
+#include <iostream>
+#include <fstream>
+#include <string>
+
+static constexpr std::string_view configFile_ = "passkey_config.toml";
+static toml::table * loadedConfig_ = nullptr;
```

Comment:
Can we use a managed pointer here ... `shared_ptr` ?

Reply:
Uses a shared_ptr now.  Also doesn't keep the configuration file loaded  anymore to safe on DRAM during runtime.